JeopardyScorekeeper/QuestionForm.cs

```diff
@@ -30,14 +30,7 @@ private void QuestionForm_Load(object sender, EventArgs e)
 
         private void btnAnswer_Click(object sender, EventArgs e)
         {
-            if(this.textBoxAnswer.Visible)
-            {
-                this.textBoxAnswer.Visible = false;
-            }
-            else
-            {
-                this.textBoxAnswer.Visible = true;
-            }
+            textBoxAnswer.Visible = !textBoxAnswer.Visible;
```

Comment:
Is it a problem that we lost the "this" in front of "textBoxAnswer"?

Reply:
Nope, the "this" is redundant in this case because the TextBox control lives in the form. VS will tell you to remove it.